src/rqt_launch/node_proxy.py

```diff
@@ -47,8 +47,8 @@ def __init__(self, parent):
     def run(self):
         while True:
             rospy.logdebug('Proc={} Died? {}'.format(
-                                               self._parent.get_proc_name(),
-                                               self._parent.has_died()))
+                self._parent.get_proc_name(),
```

Comment:
Don't break the first line and then align second with str.format

Reply:
That won't work because it would exceed 79 characters,